modules/aws/eks/variables.tf

```diff
@@ -0,0 +1,169 @@
+variable "aws_region" {
+  description = "The AWS region to build eks cluster"
+  type        = "string"
+  default     = ""
+}
+
+variable "cidr_access_eks_endpoint" {
```

Comment:
How about to rename `cidr_access_eks_https` for consistency with `cidr_access_worker_ssh`?

Reply:
sure